catalog-info.yaml

```diff
@@ -152,3 +152,138 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-auditbeat
+  description: "Auditbeat pipeline"
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/auditbeat
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: auditbeat
+      description: "Auditbeat pipeline"
+    spec:
+      #      branch_configuration: "main 7.* 8.* v7.* v8.*" TODO: temporarily commented to build PRs from forks
+      pipeline_file: ".buildkite/auditbeat/auditbeat-pipeline.yml"
+      #      maximum_timeout_in_minutes: 120 TODO: uncomment when pipeline is ready
```

Comment:
I see a few TODOs in this file, should they be uncommented? Or it's too early now?